backend/src/sms/routes/sms-campaign.routes.ts

```diff
@@ -70,6 +70,12 @@ const duplicateCampaignValidator = {
   }),
 }
 
+const tesseractCampaignValidator = {
+  [Segments.BODY]: Joi.object({
+    url: Joi.string().trim().required(),
```

Comment:
I took a look at the Joi API and turns out there is a `string.uri()` validator [1]. Perhaps we should consider using it? In particular, I think the `scheme` and `domain` options are relevant.

[1]: https://github.com/sideway/joi/blob/master/API.md#stringurioptions